py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py

```diff
@@ -26,7 +30,14 @@ def add_file(self, filename: str, filesize: int) -> None:
         self.total_size += filesize
 
     def trigger_lambda(self) -> None:
-        raise NoImplException("Cannot Trigger Lambda on a Batch")
+        try:
+            function_arn = os.environ["INGEST_FUNCTION_ARN"]
+            invoke_async_lambda(function_arn=function_arn,
+                                event=self.create_event())
+        except KeyError as e:
+            raise ArgumentException("Ingest Func Arn not Defined") from e
+        except Exception as e:
+            raise AWSException("Unable To Trigger Ingest Lambda") from e
```

Comment:
trigger this using a new s3 util that will use boto3 to do this async.

catch errors here and and raise new ones from them to improve logging.